File: MtgCardOrganizer.Dal/Entities/Cards/CardSet.cs
using System.Collections.Generic;
using MtgCardOrganizer.Dal.Entities.Collections;
using MtgCardOrganizer.Dal.Enums;

namespace MtgCardOrganizer.Dal.Entities.Cards
{
    public class CardSet : Entity
    {
        public string MultiverseId { get; set; }
        public string Artist { get; set; }
        public string Num { get; set; }
        public Rarity Rarity { get; set; }
        
        public int CardId { get; set; }
        public int SetId { get; set; }
        public Card Card { get; set; }
        public Set Set { get; set; }

        public ICollection<CardInstance> CardInstances { get; set; }
    }
}

File: frontend/src/app/card/models/card-set.ts
import { Rarity } from './rarity';
import { Card } from './card';

export class CardSet {
  id: number;

  num: string;
  setId: number;
  rarity: Rarity;
  artist: string;
  multiverseId: string;

  cardId: number;
  card: Card;

  public constructor(init?: Partial<CardSet>) {
    Object.assign(this, init);
  }
}
